__tests__/components/UserLocation.test.js

```diff
@@ -34,33 +34,29 @@ describe('UserLocation', () => {
       jest.clearAllMocks();
     });
 
-    test('renders with CircleLayers by default', (done) => {
-      const { UNSAFE_getAllByType } = render(<UserLocation />);
+    test('renders with CircleLayers by default', async () => {
+      const { UNSAFE_getAllByType } = await render(<UserLocation />);
 
-      setTimeout(() => {
-        const shapeSource = UNSAFE_getAllByType(ShapeSource);
-        const circleLayer = UNSAFE_getAllByType(CircleLayer);
+      const shapeSource = UNSAFE_getAllByType(ShapeSource);
+      const circleLayer = UNSAFE_getAllByType(CircleLayer);
 
-        expect(shapeSource.length).toBe(1);
-        expect(circleLayer.length).toBe(3);
-        done();
-      });
-    });
+      expect(shapeSource.length).toBe(1);
+      expect(circleLayer.length).toBe(3);
+    }, 1000);
```

Comment:
👍  awesome! thanks for correcting the async tests

```suggestion
    });
```

Do we need test specific timeout?! We could also lower the default timeout can't we?!

Reply:
That's right, that was left over from a different fix attempt. Good eye!